state/model.go

```diff
@@ -1214,6 +1234,8 @@ func (m *Model) destroyOps(
 				aliveEmpty++
 			default:
 				if !IsModelNotEmptyError(err) {
+					// TODO (anastasiamac 2019-4-24) we should not break out here but
```

Comment:
could we make the TODOs something like TODO(force) so we can grep for all related TODOs for a given unit of work regardless of who wrote the code?

Reply:
This violates our TODO writing convention but I like it better :) I'll change